zmon_worker_monitor/builtins/plugins/history.py

```diff
@@ -4,30 +4,43 @@
 # wrapper for kairosdb to access history data about checks
 
 import logging
+import os
+
 import requests
+import tokens
 
 from zmon_worker_monitor.builtins.plugins.distance_to_history import DistanceWrapper
 
 from zmon_worker_monitor.adapters.ifunctionfactory_plugin import IFunctionFactoryPlugin, propartial
 
+
 logger = logging.getLogger(__name__)
 
 
+# will use OAUTH2_ACCESS_TOKEN_URL environment variable by default
+# will try to read application credentials from CREDENTIALS_DIR
+tokens.configure()
+tokens.manage('uid', ['uid'])
+tokens.start()
+
+ONE_WEEK = 7 * 24 * 60 * 60
+ONE_WEEK_AND_5MIN = ONE_WEEK + 5 * 60
```

Comment:
Most of the defaults related to time filters are using 5 minutes duration. The reason for the week part could be a legacy reason that i am not fully aware of. @Jan-M might know more.